README.md

```diff
@@ -185,17 +186,17 @@ enable collaborative features as the central MLOps interface for teams.
 
 ![ZenML Architecture Diagram.](docs/book/.gitbook/assets/Scenario3.png)
 
-In case your machine is authenticated with one of the big three cloud 
-providers, this command will do the full deployment for you.
+Currently, there are two main options to deploy ZenML:
 
-```bash
-zenml deploy --provider aws  # aws, gcp and azure are supported providers
-```
+- **ZenML Cloud**: With [ZenML Cloud](https://docs.zenml.io/deploying-zenml/zenml-cloud), 
+you can utilize a control plane to create ZenML servers, also known as tenants. 
+These tenants are managed and maintained by ZenML's dedicated team, alleviating 
+the burden of server management from your end. 
 
-You can also choose to deploy with docker or helm with full control over
-the configuration and deployment. Check out the
-[docs](https://docs.zenml.io/platform-guide/set-up-your-mlops-platform/deploy-zenml)
-to find out how.
+- **Self-hosted deployment**: Alternatively, you have the flexibility to [deploy 
+ZenML on your own self-hosted environment](https://docs.zenml.io/deploying-zenml/zenml-self-hosted). 
+This can be achieved through various methods, including using our CLI, Docker, 
+Helm, or HuggingFace Spaces.
```

Comment:
Also soon mlstacks separately as well.

Reply:
For now, I would not mention it here though.